_posts/en/newsletters/2019-10-23-newsletter.md

```diff
@@ -0,0 +1,130 @@
+  allowing implementers to identify flaws or sub-optimal requirements in
+  the current proposals that might be missed by people who only read the
+  documentation.
+
+    The ultimate goal of the review club is to allow participants to
+    gain enough technical familiarity with the proposals to be able to
+    either vocally support the proposals, advocate for changes to the
+    proposals, or lucidly explain why the proposals shouldn't be adopted
+    into the Bitcoin consensus rules.  Changing Bitcoin is something
+    that should be done carefully, and it's in every user's interest
+    that a large number of technical reviewers examine the proposals for
+    possible flaws before they are implemented and before users are
+    asked to consider upgrading their full nodes to enforce the new
+    rules---a change that can't be safely undone for as long as the
+    rules affect anyone's bitcoins.  Whether through the review club or
+    in some other way, Optech strongly encourages all technically
+    skilled Bitcoin users to dedicate time to reviewing the taproot set
+    of proposals.
+
+    Anyone wanting to participate should RSVP soon so that the
+    organizers can estimate the total number of participants and start
+    forming study groups.  To register or learn more, please see
+    [FIXME][].
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin wallets
+and services.*
+
+- **Electrum Lightning support:** In a [series of commits this
+  month][electrum commits], Electrum has merged support for Lightning
+  Network into master. A presentation deck titled [Lightning Implementation in
+  Electrum][electrum lightning presentation] by Thomas Voegtlin provides some
+  background information and screenshots.
+
+- **Blockstream Green Tor support:** Version 3.2.4 of the Blockstream Green
+  Wallet [adds built-in Tor support][blockstream green tor article] for both iOS and Android.
+  While Tor was supported in previous Android versions, it required a separate
+  application whereas both Android and iOS versions are now bundled with Tor
+  support.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [LND #3696][] raises the default maximum CLTV expiry from 1,008 blocks
+  (about 1 week) to 2,016 blocks (about two weeks).  This is the maximum
+  amount of time a new payment will be valid before it can be reclaimed
+  by its spender if it hasn't been accepted by the receiver.  LND
+  recently tried to keep this at 1,008 blocks by decreasing the CLTV
+  delta---the amount of time each routing node along the payment path
+  has to claim its particular payment---from 144 blocks to 40 blocks
```

Comment:
I think it would be clearer to explicitly say that the delta is the value that the CLTV is decremented in each hop.

Reply:
That would be a good description of how it works but I feel the current description better communications what it accomplishes---assures that you have a minimum of x blocks in which to claim your payment.  The value is important because, if your node is offline for more than x blocks, someone might be able to steal from you.